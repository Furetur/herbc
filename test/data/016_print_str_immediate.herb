fn main() {
    print("Hello, world! 123 Привет ♥");
}