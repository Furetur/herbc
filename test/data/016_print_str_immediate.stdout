Hello, world! 123 Привет ♥